### PR TITLE
leo's lab meeting function - plotting by date

### PR DESCRIPTION
**Pull request by leoscholl**

### CODE REVIEW

File: aopy/visualization.py [hunk 1]
```diff
@@ -413,3 +412,69 @@ def plot_trajectories(trajectories, bounds=None, ax=None):
...
+
+def test_function(arg1, arg2, param1='foobar'):
+    '''
+    [summary]
+
+    Args:
+        arg1 ([type]): [description]
+        arg2 ([type]): [description]
+        param1 (str, optional): [description]. Defaults to 'foobar'.
+    '''
```
Comment (leoscholl) [inline aopy/visualization.py:480]:
```suggestion
```

File: aopy/visualization.py [hunk 2]
```diff
@@ -413,3 +412,69 @@ def plot_trajectories(trajectories, bounds=None, ax=None):
...
+            ax.set_ylabel('weight (kg)')
+
+        ..image: _images/columns_by_date.png
+
+    Args:
+        df (pd.DataFrame): dataframe with 'date' column
+        *columns (str): dataframe column names to plot
+        method (str, optional): how to combine data within a single date. Can be 'sum' or 'mean'.
+        ax (pyplot.Axes, optional): axis on which to plot
+    '''
```
Comment (m-nolan) [inline aopy/visualization.py:442]: consider adding figure/axes handles as return values :)

**Review by m-nolan — APPROVED** [review]
Review figure handle outputs, resolve later. Otherwise, A OK

**Review by resatomo — APPROVED** [review]
Sounds good

**Pull request merged**